src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25256.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue25256 : _IssuesUITest
+	{
+		public Issue25256(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "ListView ScrolledEventArgs.ScrollY does not reset when ItemsSource changes";
+
+		[Test]
+		[Category(UITestCategories.ListView)]
+		public void ListViewShouldHaveProperScrollYAfterResetingItemsSource()
+		{
+			App.WaitForElement("button");
+			App.ScrollDown("list", ScrollStrategy.Gesture, swipePercentage: 0.5, swipeSpeed: 100);
+			App.Click("button");
+
+			var label = App.FindElement("scrollYLabel").GetText();
+			Assert.That(label, Is.EqualTo("0"));
```

Comment:
The test is failing on iOS:
<img width="771" alt="image" src="https://github.com/user-attachments/assets/6c17edf6-623f-4253-b794-eec65a0fec4c">

Reply:
It should work now